File: Practice/src/introduction_coding_tests/SplitAndSaveAsArray.py
def solution(my_str, n):
    return [my_str[i * n:(i + 1) * n] for i in range(int((len(my_str) + n - 1) / n))]
    # return [my_str[i: i + n] for i in range(0, len(my_str), n)]

File: Practice/src/introduction_coding_tests/SplitAndSaveAsArray.java
package introduction_coding_tests;

import java.util.stream.IntStream;

public class SplitAndSaveAsArray {
//	잘라서 배열로 저장하기
	public String[] solution(String my_str, int n) {
		String[] answer = new String[(my_str.length() + 5) / 6];
		IntStream.range(0, my_str.length()).forEach(i -> answer[(i - 1) / 6] += Character.valueOf(my_str.charAt(i)));
		return answer;
	}
}